wp-content/plugins/jetpack/_inc/build/contact-form/js/grunion.min.js

```diff
@@ -0,0 +1 @@
b-new-options").append('<div id="fb-option-box-'+r+'" class="fb-new-fields"><span optionid="'+r+'" class="fb-remove-option"></span><label></label><input type="text" id="fb-option'+r+'" optionid="'+r+'" value="'+GrunionFB_i18n.optionLabel+'" class="fb-options" /><div>'),jQuery("#fb-new-field"+i+" .fb-fields").append('<div id="fb-checkbox-multiple-'+i+"-"+r+'"><input type="checkbox" disabled="disabled" id="fb-field'+i+'" name="checkbox-multiple-'+i+'" /><span>'+GrunionFB_i18n.optionLabel+'</span><div class="clear"></div></div>')):(jQuery("#fb-new-options").append('<div id="fb-option-box-'+r+'" class="fb-new-fields"><span optionid="'+r+'" class="fb-remove-option"></span><label></label><input type="text" id="fb-option'+r+'" optionid="'+r+'" value="" class="fb-options" /><div>'),jQuery("#fb-field"+i).append('<option id="fb-'+i+"-"+r+'" value="'+i+"-"+r+'"></option>')),e.fields[i].options[r]="",jQuery("#fb-option"+r).focus().select()}catch(e){}}()})),y.on("keyup",".fb-options",(function(){!function(i){try{var n=jQuery("#fb-field-id").val(),t=i.attr("optionid"),o=i.val(),r=jQuery("#fb-new-type").val();"radio"===r?jQuery("#fb-radio-"+n+"-"+t+" span").text(o):"checkbox-multiple"===r?jQuery("#fb-checkbox-multiple-"+n+"-"+t+" span").text(o):jQuery("#fb-"+n+"-"+t).text(o),e.fields[n].options[t]=o}catch(e){}}(jQuery(this))})),y.on("click",".fb-remove-option",(function(){!function(i){try{var n=jQuery("#fb-field-id").val(),t=jQuery("#fb-option"+i).val(),o=jQuery("#fb-new-type").val();jQuery("#fb-option-box-"+i).remove(),"radio"===o?jQuery("#fb-radio-"+n+"-"+i).remove():"checkbox-multiple"===o?jQuery("#fb-checkbox-multiple-"+n+"-"+i).remove():jQuery("#fb-"+n+"-"+i).remove();var r=e.fields[n].options.indexOf(t);-1!==r&&e.fields[n].options.splice(r,1)}catch(e){}}(jQuery(this).attr("optionid"))})),jQuery("#tab-preview a").click((function(){return p("preview"),!1})),jQuery("#fb-prev-form").click((function(){return p("preview"),!1})),jQuery("#tab-settings a").click((function(){return p(),!1})),jQuery("#fb-field-my-email").blur((function(){!function(){try{var i=jQuery("#fb-field-my-email").val();e.to=i}catch(e){}}()})),jQuery("#fb-field-subject").blur((function(){!function(){try{var i=jQuery("#fb-field-subject").val();e.subject=i}catch(e){}}()})),y.on("mouseenter",".fb-form-case .fb-new-fields",(function(){f("show",jQuery(this))})),y.on("mouseleave",".fb-form-case .fb-new-fields",(function(){return f("hide"),!1})),y.on("click",".fb-edit-field",(function(){return l(jQuery(this)),!1})),y.on("click",".fb-edit-field .fb-reorder",(function(){return!1})),y.on("click","#fb-save-field",(function(){return c(),b(),!1})),jQuery("#fb-feedback").click((function(){var e=jQuery(this).attr("href");return window.parent.location=e,!1})),jQuery("#sortable").sortable({axis:"y",handle:".fb-reorder",revert:!0,start:function(){jQuery(".fb-edit-field").hide()}}),jQuery("#draggable").draggable({axis:"y",handle:".fb-reorder",connectToSortable:"#sortable",helper:"clone",revert:"invalid"})}}}();
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).